README.md

```diff
@@ -72,3 +78,18 @@ Run the docker container:
 ```
 docker run -p 8001:8001 -e SPRING_DATASOURCE_PASSWORD=yourPasswordHere --rm mod-ldp
 ```
+
+## Additional information
+
+### Other documentation
+
+[Library Data Platform](https://github.com/library-data-platform/ldp) (LDP)
+-- an open source platform for analytics in libraries.
+
+[ui-ldp](https://github.com/library-data-platform/ui-ldp)
+-- LDP query builder UI for FOLIO/ReShare
+
+### Issue tracker
+
+The project use this GitHub [issue tracker](https://github.com/library-data-platform/mod-ldp/issues).
```

Comment:
@dcrossleyau I just noticed this typo "The project use" (instead of "uses"). Easy to fix here, of course, but I wonder whether it might be in your template, and therefore might have sneaked into a lot of projects?

Reply:
No, it is a plain old typo. I will fix it. Thanks eagle-eyes.